### ISSUE TITLE
Random characters across the website

### ISSUE THREAD

**Issue by carterlando** [clayharmon/bb-lazy-load#11]
Hi,

Activating this plugin makes random characters to appear across the website, even with none of the settings enabled. Deactivating the plugin fixes the issue.

Do you have any idea what could cause this and is there any way to fix it?

![image](https://user-images.githubusercontent.com/27959710/58074862-f109fb80-7bae-11e9-983a-976b30da31be.png)

Later edit:
I did some debugging and it also happens on the 2019 WP Theme with only Beaver Builder Lazy Load plugin and Beaver Builder itself. Disabling any of these 2 plugins fixes the issue. The PHP version is 7.2.9

![image](https://user-images.githubusercontent.com/27959710/58078022-7218c100-7bb6-11e9-8874-8cc0f9ba9e41.png)

**Comment by carterlando** [comment]
I managed to fix it for me by updating line 133 in bb-lazy-load.php 
from
`$doc->loadHTML($content);`

to
`$doc->loadHTML('<?xml encoding="utf-8" ?>' . $content);`

I got my solution from here:
https://stackoverflow.com/questions/8218230/php-domdocument-loadhtml-not-encoding-utf-8-correctly

I'm not sure if this is the correct solution all-round but it would be great if this could be solved in a new update.

Also, I see all content is filtered, it would be great if only pages built with beaver builder could be targeted by the plugin.

Thanks.
